clr-installer/main.go

```diff
@@ -268,7 +245,49 @@ func execute(options args.Args) error {
 	if options.AllowInsecureHTTPSet {
 		md.AllowInsecureHTTP = options.AllowInsecureHTTP
 	}
+}
+
+func processPamSaltOption(options args.Args) error {
+	if options.PamSalt != "" {
+		if status, err := user.IsValidPassword(options.PamSalt); !status {
+			return fmt.Errorf(err)
+		}
+
+		hashed, errHash := encrypt.Crypt(options.PamSalt)
+		if errHash != nil {
+			return errHash
+		}
+
+		fmt.Println(hashed)
+		return nil
+	}
+
+	return nil
+
+}
+
+func processStubImageOption(options args.Args, md *model.SystemInstall) error {
```

Comment:
This might be better named 
```suggestion
func processNotStubImageOption(options args.Args, md *model.SystemInstall) error {
```

Reply:
Sounds good. Done